shared/settings/feedback/container.desktop.tsx

```diff
@@ -18,9 +18,10 @@ const mapDispatchToProps = dispatch => ({
   onSendFeedback: (feedback, sendLogs) => dispatch(SettingsGen.createSendFeedback({feedback, sendLogs})),
 })
 
-const mergeProps = (s, d, o) => ({
+const mergeProps = (s, d, o: OwnProps) => ({
   ...s,
   ...d,
+  feedback: o.routeProps.get('feedback'),
```

Comment:
I think `OwnProps.routeProps` is going away. Should use `getRouteProps` instead https://github.com/keybase/client/blob/49eb317d732f264a69a65ac45ac33c4fc6be3d7a/shared/fs/container.tsx#L37